.github/actions/test-and-report/action.yml

```diff
@@ -142,7 +133,24 @@ runs:
       with:
         name: ${{ steps.name_gen.outputs.REPORT_NAME }}
         path: ${{ inputs.test_directory }}/reports/test-report.html
-        retention-days: 30
+        retention-days: 7
+      continue-on-error: true
+
+    - name: Publish Test Summary
```

Comment:
I think this and the following will both show up on the test run, just with skipped status if the conditional is met, so perhaps lets change the names of these steps to differentiate between them, so the reader isn't confused why one ran and what appears like a duplicate of same step was also skipped